astrapy/serverless.py

```diff
@@ -114,62 +151,23 @@ def delete_subdocument(self, id, subdoc):
                 "update": {"$unset": {subdoc: ""}},
             }
         }
-        return self.astra_client.request(
-            method=http_methods.POST,
-            path=f"{self.base_path}",
-            json_data=json_query,
-        )
-
-    def delete(self, id):
-        json_data = {"deleteOne": {"filter": {"_id": id}}}
-        return self.astra_client.request(
-            method=http_methods.POST, path=f"{self.base_path}", json_data=json_data
-        )
-
-    def insert_many(self, documents=None):
-        json_query = {"insertMany": {"documents": documents}}
-
         return self.astra_client.request(
             method=http_methods.POST, path=f"{self.base_path}", json_data=json_query
         )
 
-    def push(self, id, update, options):
-        json_data = {
-            "findOneAndUpdate": {
-                "filter": {"_id": id},
-                "update": update,
-                "options": options,
-            }
-        }
-        res = self.astra_client.request(
-            method=http_methods.POST,
-            path=f"{self.base_path}",
-            json_data=json_data,
-        )
-        return res.get("data")
-
-    def pop(self, id, update, options):
-        json_data = {
-            "findOneAndUpdate": {
-                "filter": {"_id": id},
-                "update": update,
-                "options": options,
-            }
-        }
-        res = self.astra_client.request(
+    def insert_many(self, documents=None, id_path=""):
+        return self.astra_client.request(
```

Comment:
Doesn't look like we have any tests for `documents=None` - should we either add some or make it a required param?

Reply:
Considering i think the behavior of inserting with documents=None is a little confusing, i think i'll make it required for now!